content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,17 @@
+## Bug fixes and minor enhancements
+
+- Add a precise SHA reference to the release manifest ([Pull Request 14959](https://github.com/istio/istio/pull/14959))
+- Fix regression caused by forward port merge error ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
+- Add citadel replicaCount flag back and workloadCertTtl to support multi-citadel and short cert TTL for perf test ([Pull Request 15066](https://github.com/istio/istio/pull/15066))
+- Fix Mixer CRD store issue ([Issue 14841](https://github.com/istio/istio/issues/14841))
+- Add HTTP/1.0 support to gateway configuration generation ([Pull Request 15062](https://github.com/istio/istio/pull/15062))
+- Redis filter:
```

Comment:
May want to mention it is behind a feature flag

Reply:
```suggestion
- Redis filter (this feature is guarded with the feature flag PILOT_ENABLE_REDIS_FILTER, disabled by default):
```